zvariant_derive/src/dict.rs

```diff
@@ -5,7 +5,7 @@ use zvariant_utils::macros;
 
 use crate::utils::*;
 
-fn dict_name_for_field(
+pub(crate) fn dict_name_for_field(
```

Comment:
It's smart using the existing code we're not using it for a dict, so either this should be moved to utils and renamed accordingly or we shouldn't touch this and use `rename_identifier` instead.

Reply:
Switched to rename identifier